1.0/gradle/vividus-test.gradle

```diff
@@ -134,6 +141,14 @@ tasks.withType(StoriesExec).configureEach {
     }
     ignoreExitValue true
     doLast {
+        def pathToExitCodeFile = project.hasProperty('fileToSaveExitCode') ? new groovy.text.SimpleTemplateEngine().createTemplate(project.fileToSaveExitCode)
+                                                                                                                   .make(project.getProperties())
+                                                                                                                   .toString()
```

Comment:
- why not to use [`findProperty`](https://discuss.gradle.org/t/getproperty-vs-findproperty/21822/2)? 
- why is all logic around templating needed?

Reply:
`why not to use findProperty?
why is all logic around templating needed?`
to resolve a references
./gradlew dS  -PfileToSaveExitCode='${project.buildDir}/exitCode.txt' -p vividus-tests

Reply:
we shouldn't disclose gradle runtime properties in the input parameters, add an extra flag:
```
./gradlew dS -PfileToSaveExitCode=/exitCode.txt -PresolvePathAgainstProjectBuildDir=true -p vividus-tests
```